pkg/lib/ssh-add-key.sh

```diff
@@ -2,8 +2,13 @@
 
 set -euf
 
+p=$2
+if test -z "$p"; then
+  p=authorized_keys
+fi
+
 d=$HOME/.ssh
-f=$d/authorized_keys
+f=$d/$p
```

Comment:
since `$p` could have spaces here, I think it needs `"`.

Reply:
No, try it. :-)